base/math.jl

```diff
@@ -74,15 +74,15 @@ macro evalpoly(z, p...)
         ai = symbol("a", i)
         push!(as, :($ai = $a))
         a = :(muladd(r, $ai, $b))
-        b = :(muladd(-s, $ai, $(esc(p[i]))))
+        b = :($(esc(p[i])) - s * $ai) # don't have efficient multiply-subtract
```

Comment:
See @yuyichao's recent email, stating that this might be a function of the optimization level (fast vs. slow instruction selector in LLVM). I'd add a comment pointing to the discussion, and explaining that this is at least LLVM-version (and probably also platform) dependent.

Maybe it's still too early for this judgement.